Resources/doc/Getting-started.rst

```diff
@@ -0,0 +1,129 @@
+| **user\_groups.yml**   | Define all user groups (code + label)                                   |
++------------------------+-------------------------------------------------------------------------+
+| **user\_roles.yml**    | Define all user roles (code + label)                                    |
++------------------------+-------------------------------------------------------------------------+
+| **users.yml**          | Define users list                                                       |
++------------------------+-------------------------------------------------------------------------+
+
+You can still have a look on the `Akeneo PIM minimal
+fixtures <https://github.com/akeneo/pim-community-dev/tree/master/src/Pim/Bundle/InstallerBundle/Resources/fixtures/minimal>`__
+set to get a full list of the files and their expected format.
+
+EE edition (incl. CE files)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+EE edition adds support of ACL, please refer to the minimal fixtures set
+located in the InstallerBundle to see the structure and the content of
+the files. Note that you cannot define ACL in the init.xslx file and
+will have to define them in seperatly
+
++---------------------------------------+-------------------------------------+
+| File                                  | Description                         |
++=======================================+=====================================+
+| **attribute\_groups\_accesses.yml**   | Contains ACL for attribute groups   |
++---------------------------------------+-------------------------------------+
+| **category\_accesses.yml**            | Contains ACL for categories         |
++---------------------------------------+-------------------------------------+
+| **locale\_accesses.yml**              | Contains ACL for locales            |
++---------------------------------------+-------------------------------------+
+
+Customize init.xslx !
+~~~~~~~~~~~~~~~~~~~~~
+
+Edit the init.xlsx file to your needs following the instructions inside
+the file itself. The file is composed of various tabs that allow you to
+set your catalog structures : 
+
+- channels 
+- categories
+- group types 
+- association types
+- attributes 
+- attribute groups
+- attribute options
+- families (as many tabs as required)
+
+See `tab description
+section <Home.rst#define-the-structure-of-your-catalog>`__ for more
+details on how to customize the init.xslx file.
+
+Change PIM parameter to use your custom installation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code:: yml
+
+    parameters:
+        pim_installer.fixture_loader.job_loader.config_file: 'PimExcelConnectorBundle/Resources/config/fixtures_jobs.yml'
+
+
+Define the data used by the installer :
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code:: yml
+
+    # app/config/pim_parameters.yml
+    parameters:
+        ...
+        installer_data: 'AcmeDemoBundle:minimal'
```

Comment:
@fitn I didn't understand :/ the parameters.yml.dist already contains that statement so it is just about changing it right ?